### PR TITLE
[docs] Updating contribution info

### PR DESCRIPTION
**Pull request by gunnarmorling**
To be merged after the release is out

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@rk3rn3r, as you just were mentioning merge commits: added a few lines to the contributor's guide. Can you merge if it's ok?

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -270,3 +270,11 @@ Here's a quick check list for a good pull request (PR):
 * New/changed features have been documented
 * A full build completes successfully
 * Do a rebase on upstream `master`
+
+### PR Handling (For Committers)
+
+* Don't merge your own PRs, ensure four eyes principle (unless it's trivial typo fixes and the like)
+* Ensure there are no "Octocat" commits
```
Comment (rk3rn3r) [inline CONTRIBUTE.md:277]: Does everyone understand "Octocat commits"?

Comment (gunnarmorling) [reply CONTRIBUTE.md]: Good point; made that a link to GH docs.

**Comment by gunnarmorling** [comment]
Applied.

**Pull request merged**